.eslintrc.js

```diff
@@ -1,11 +1,10 @@
 module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint', 'cypress'],
```

Comment:
Is removing `@typescript-eslint` necessary?

Reply:
Good catch! Fixed